azure-pipelines.yml

```diff
@@ -43,23 +46,27 @@ stages:
         projectKey: '2011-fakebook-project3_posts'
         extraProperties: |
           sonar.cs.opencover.reportsPaths=$(Agent.TempDirectory)/**/coverage.opencover.xml
-          sonar.cs.vstest.reportsPaths=$(Agent.TempDirectory)/*.trx
+
     - task: UseDotNet@2
       displayName: 'Use .NET 5 SDK'
       inputs:
-        packageType: 'sdk'
-        version: '5.x'
-        installationPath: $(Agent.ToolsDirectory)/dotnet
-        workingDirectory: 'Fakebook.Posts.RestApi/'
+        workingDirectory: '$(Agent.ToolsDirectory)/dotnet'
```

Comment:
??? workingDirectory is one input here that you don't need, and packageType and version the two that you _do_ need